layouts/css/base.styl

```diff
@@ -134,6 +134,11 @@ pre
     article
         margin-left 220px
 
+.edit-link
+    float right
+    font-size 0.6em
+    transform translateY(30%)
```

Comment:
Leaving this hack until there's a decision on the appearance and position of "Edit on GitHub" link.